src/main/java/org/galatea/starter/entrypoint/IexRestController.java

```diff
@@ -48,4 +50,31 @@
     return iexService.getLastTradedPriceForSymbols(symbols);
   }
 
+  /**
+   * Get the historical price data for a symbol over a range of dates
+   *
+   * @param symbol the symbol to get the price data for.
+   * @param range the range option.  See "https://iexcloud.io/docs/api/#historical-prices" for
+   *     options.
+   * @param date the date to get the data from.  YYYYMMDD format.
+   * @return a historical price object for the symbol passed in.
+   */
+  @GetMapping(value = "${mvc.iex.getHistoricalPricePath}",
+      produces = {MediaType.APPLICATION_JSON_VALUE})
+  public List<IexHistoricalPrice> getHistoricalPrice(
+      @RequestParam(value = "symbol", required = false) final String symbol,
+      @RequestParam(value = "range", required = false) final String range,
+      @RequestParam(value = "date", required = false) final String date) {
+    if (symbol == null) {
+      return Collections.emptyList();
```

Comment:
Add logs around each of these if branches, explaining what happened to the request